### ISSUE TITLE
It's not that much an issue

### ISSUE THREAD

**Issue by fredericplante** [dgtlmoon/uCamII#1]
I was wondering if this UCamII from 4dsystem is in fact an OV528 serial camera?

**Comment by dgtlmoon** [comment]
Yes, I believe it may be the same but assembled in Australia with a slightly different (improved) circuit board

The instructions certainly seem the same

https://yadom.fr/downloadable/download/sample/sample_id/31/
http://www.4dsystems.com.au/productpages/uCAM-II/downloads/uCAM-II_datasheet_R_1_4.pdf

but the PCB layout is a little different
